lib/readline.js

```diff
@@ -921,6 +921,9 @@ exports.Interface = Interface;
 const KEYPRESS_DECODER = Symbol('keypress-decoder');
 const ESCAPE_DECODER = Symbol('escape-decoder');
 
+let timeoutRef;
+const ESCAPE_CODE_TIMEOUT = 100;
```

Comment:
I would prefer a good default. Why would you ever want to tune this differently?